scaffolds/Lite/package.json

```diff
@@ -5,50 +5,38 @@
   "engines": {
     "node": ">=8.0.0"
   },
-  "files": [
-    "src/",
-    "build/",
-    "public/",
-    "tests/",
-    "_gitignore",
-    "_editorconfig",
-    "_eslintignore",
-    "_eslintrc.js",
-    "_stylelintignore",
-    "_stylelintrc.js",
-    "ice.config.js"
-  ],
   "scripts": {
-    "start": "ice-scripts dev",
-    "build": "ice-scripts build",
-    "prepublishOnly": "npm run build & npm run screenshot",
-    "lint": "npm run eslint && npm run stylelint",
-    "eslint": "eslint --cache --ext .js,.jsx ./",
-    "stylelint": "stylelint ./**/*.scss",
-    "screenshot": "screenshot -l"
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "lint": "node_modules/eslint/bin/eslint.js src",
```

Comment:
`react-scripts` does not support the `react-scripts lint` command. References:
1. [Run Lint from the command line?](https://github.com/facebook/create-react-app/issues/1217)
2. [added react-scripts lint #2729](https://github.com/facebook/create-react-app/pull/2729)

Reply:
just `eslint .js,.jsx ./`, npm will find in node_modules first.